mission_control/test/test_jaus_ros_bridge_interface_mission_control.py

```diff
@@ -0,0 +1,132 @@
+import math
+import rosnode
+import rospy
+import rostest
+from mission_control.msg import LoadMission
+from mission_control.msg import ExecuteMission
+from mission_control.msg import ReportExecuteMissionState
+from mission_control.msg import ReportLoadMissionState
+from mission_control.msg import AbortMission
+from mission_control.msg import QueryMissions
+from mission_control.msg import RemoveMissions
+from mission_control.msg import ReportMissions
+
+
+class TestJausRosBridgeInterface(unittest.TestCase):
+    """ 
+        Simulate messages sent by the jaus ros bridge
+    """
+
+    rospy.init_node('thruster_response_if_system_fails')
+    mission_execute_state = ReportExecuteMissionState.PAUSED
+    mission_load_state = False
+    dir_path = os.path.dirname(os.path.abspath(__file__)) + '/test_files/'
+
+    mission_load_state_flag = False
+    report_mission_flag = False
+    report_mission = ReportMissions()
+
+    def callback_mission_execute_state(self, msg):
+        self.report_mission = msg
+        print(msg)
+
+    def callback_report_mission(self, msg):
+        self.report_mission = msg
+        self.report_mission_flag = True
+
+    def callback_mission_load_state(self, msg):
+        self.mission_load_state = msg.load_state
+        self.mission_load_state_flag = True
+
+    def test_jaus_ros_bridge_interface(self):
+        rospy.Subscriber('/mission_control_node/report_mission_execute_state',
+                         ReportExecuteMissionState, self.callback_mission_execute_state)
+
+        rospy.Subscriber('/mission_control_node/report_mission_load_state',
+                         ReportLoadMissionState, self.callback_mission_load_state)
+
+        rospy.Subscriber('/mission_control_node/report_missions',
+                         ReportMissions, self.callback_report_mission)
+
+        simulated_mission_control_load_mission_pub = rospy.Publisher('/mission_control_node/load_mission',
+                                                                     LoadMission, latch=True, queue_size=1)
+
+        simulated_mission_control_execute_mission_pub = rospy.Publisher(
+            '/mission_control_node/execute_mission', ExecuteMission, latch=True, queue_size=1)
+
+        simulated_abort_mission_msg_pub = rospy.Publisher(
+            '/mission_control_node/abort_mission', AbortMission, latch=True, queue_size=1)
+
+        simulated_query_mission_msg_pub = rospy.Publisher(
+            '/mission_control_node/query_missions', QueryMissions, latch=True, queue_size=1)
+
+        simulated_remove_mission_msg_pub = rospy.Publisher(
+            '/mission_control_node/remove_missions', RemoveMissions, latch=True, queue_size=1)
+
+        # TEST - Load a wrong mission
+        mission_to_load = LoadMission()
+        mission_to_load.mission_file_full_path = "NO_MISSION"
+        simulated_mission_control_load_mission_pub.publish(mission_to_load)
+        while (not rospy.is_shutdown() and self.mission_load_state_flag == False):
```

Comment:
@patapufate nit: equality comparison for booleans is non-idiomatic. Consider something like: 
```suggestion
        while (not rospy.is_shutdown() and not self.got_mission_load_state):
```

Reply:
Also, parenthesis in Python control flow statements are redundant. Here and everywhere else.

Reply:
fixed